doc/orchestrator.rst

```diff
@@ -165,3 +165,460 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+
+Lets check to make sure the database tensors do not exist in the incorrect databases:
+
+.. code-block:: python
+
+  check_single_shard_db_tensor_incorrect = driver_client_single_shard.key_exists("tensor_2")
+  check_multi_shard_db_tensor_incorrect = driver_client_multi_shard.key_exists("tensor_1")
+  logger.info(f"The multi shard array key exists in the incorrect database: {check_single_shard_db_tensor_incorrect}")
+  logger.info(f"The single shard array key exists in the incorrect database: {check_multi_shard_db_tensor_incorrect}")
+
+When you run the experiment, the following output will appear:
+
+.. code-block:: bash
+
+  00:00:00 system.host.com SmartSim[#####] INFO The multi shard array key exists in the incorrect database: False
+  00:00:00 system.host.com SmartSim[#####] INFO The single shard array key exists in the incorrect database: False
+
+Initializing the colo Model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In the next stage of the experiment, we will utilize the
+application script by configuring and creating
+a SmartSim colocated ``Model``.
+
+Step 1: Configure
+"""""""""""""""""
+You can specify the run settings of a model.
+In this example, we invoke the Python interpreter to run a
+python script. To specify this, we use the ``Experiment.create_run_settings()`` function.
+This function returns a ``RunSettings`` object.
+Here, we specify the path to the source code file (script) to
+``exe_args`` and the run command to ``exe``.
+
+.. code-block:: python
+
+  model_settings = exp.create_run_settings(exe="python", exe_args="./model_multidb_example.py")
+
+With the ``RunSettings`` instance,
+configure the the distribution of computational tasks (``RunSettings.set_nodes()``) and the number of instances
+the script is execute on each node (``RunSettings.set_tasks_per_node()``). In this
+example, we specify to SmartSim that we intend to execute the script once on a single node.
+
+.. code-block:: python
+
+  model_settings.set_nodes(1)
+  model_settings.set_tasks_per_node(1)
+
+Step 2: Initialize
+""""""""""""""""""
+Next, create a ``Model`` instance using the ``Experiment.create_model()``.
+Pass the ``model_settings`` object as an argument
+to the ``create_model()`` function and assign to the variable ``model``.
+
+.. code-block:: python
+
+  model = exp.create_model("colocated_model", model_settings)
+
+Step 2: Colocate
+""""""""""""""""
+To colocate the model, use the ``Model.colocate_db_tcp()`` function to
+Colocate an Orchestrator instance with this Model over TCP/IP.
+
+.. code-block:: python
+
+  model.colocate_db_tcp()
+
+This method will initialize settings which add an unsharded
+database to this Model instance. Only this Model will be able
+to communicate with this colocated database by using the loopback TCP interface.
+
+Step 3: Start
+"""""""""""""
+Next, launch the colocated model instance using the ``Experiment.start()`` function.
+
+.. code-block:: python
+
+  exp.start(model, block=True, summary=True)
```

Comment:
I think there's a logical problem in the example. Until this start() method is called, the databases aren't launched, so the code we were looking at earlier will fail. We need to present this in order so that people can test and run each step as they go

Reply:
At the very least, I think you need to explain that you're presenting things in a logical order but that the run is differently ordered

Reply:
hmmm I am a little confused by this comment, lets discuss during out 1:1!

Reply:
add comments to the top of the app script section: the experiment must have been started to launch the orch used within the script

Reply:
otherwise it will fail to connect

Reply:
disclaimer